src/charm.py

```diff
@@ -162,9 +166,11 @@ def _on_server_cert_changed(self, _):
         self._update_mimir_cluster()
 
     def _on_mimir_cluster_changed(self, _):
+        self.nginx.configure_pebble_layer(tls=self._is_tls_ready)
```

Comment:
I'm a bit confused: we use `_is_tls_ready` for nginx together with `self.ingress` from traefik?

Reply:
I guess it's similar to how we don't use TLS only on traefik in COS Lite, and we also configure the workloads to use it